modules/user-guide/pages/installing-vale-cli.adoc

```diff
@@ -26,7 +26,7 @@ On the Vale site, click the tab for your operating system.
 .The `$HOME/.vale.ini` file
 [source,ini]
 ----
-StylesPath = styles
+StylesPath = .vale/styles
```

Comment:
I would suggest being consistent with https://vale.sh/hub/redhat/, which is consistent with other configurations in the Hub. Therefore, keep:

```suggestion
StylesPath = styles
```

Reply:
* `StylePath = styles` is consistent with https://vale.sh/hub/ and https://vale.sh/generator/. I believe we should use it as default.

* `StylePath = .vale/styles` denotes a configuration following previous recommendations in the Vale project documentation. See https://vale.sh/docs/topics/config/

* `StylePath = .github/styles` denotes a configuration following what Vale GitHub action documentation requires. See https://github.com/errata-ai/vale-action

Reply:
@themr0c - Reverted 'StylesPath' back to the default configuration as per suggestion. Thanks for clarifying!